src/javascript/app/pages/user/update_balance.js

```diff
@@ -18,14 +20,25 @@ const updateBalance = (response) => {
 
         const updateBalanceByAccountId = (account_id, updated_balance, account_currency) => {
             const el_balance_span = document.querySelector(`.account__switcher-balance-${account_id}`);
+            const reset_button      = createElement('button', { text: localize('Reset balance'), class: 'account__switcher-balance btn btn--secondary btn__small reset_btn' });
+
+            const is_virtual = /^VRT/.test(account_id);
+            const is_current = Client.get('loginid') === account_id;
 
             if (el_balance_span) {
                 const display_balance = formatMoney(account_currency, updated_balance);
 
                 el_balance_span.innerHTML = display_balance;
-                
-                const is_virtual = /^VRT/.test(account_id);
-                const is_current = Client.get('loginid') === account_id;
+
+                // show reset button
+                if (updated_balance !== 10000 && is_virtual && is_current) {
```

Comment:
What if the balance is bigger than 10000? Should it show reset button?

Reply:
Yes. Only if balance is not equals to 10000 it will **not** show the reset button.